homeassistant/components/github/coordinator.py

```diff
@@ -29,6 +29,9 @@
           message: messageHeadline
           url
           sha: oid
+          status: status {
```

Comment:
```suggestion
          status: statusCheckRollup {
```
this also includes actions